pallets/settlement/src/lib.rs

```diff
@@ -494,6 +519,31 @@ decl_module! {
             Ok(())
         }
 
+        /// Registers a new venue for confidential transfers.
+        ///
+        /// * `details` - Extra details about a venue
+        /// * `signers` - Array of signers that are allowed to sign receipts for this venue
+        /// * `venue_type` - Type of venue being created
+        ///
+        /// # Weight
+        /// `200_000_000 + 5_000_000 * signers.len()`
+        #[weight = 200_000_000 + 5_000_000 * u64::try_from(signers.len()).unwrap_or_default()]
+        pub fn create_confidential_venue(origin, details: VenueDetails, signers: Vec<T::AccountId>, venue_type: VenueType) -> DispatchResult {
```

Comment:
I think we don't even need this as `create_venue()` do achieve the same functionality.